app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,10 @@
+<% provide(:title, t("about")) %>
```

Comment:
1. Em có thể lượt bỏ dấu ngoặc của `provide` nhé -> `<% provide :title, t("about") %>`
2. thêm space sau dấu `=` và `%>` nhé
-> `<%= t("about") %>`
3. Fix các file bị tương tự
4. Ngoài ra, hãy sử dụng I18n cho toàn bộ file đi em :p 
5. Dùng tag của rails như `link_to` thay vì `a`